src/01-functions.js

```diff
@@ -131,7 +153,13 @@ function compareTypes() {}
  *  flipSign(50);
  *  //> -50
  */
-function flipSign() {}
+function flipSign(a) {
+  if (a === Number(-a)) {
```

Comment:
This works but consider is your intention clear to other developers. Maybe there are more straight forward ways we could we could make this sign flip. Hint: multiplying by a negative.